Tests/ML/test_download_upload.py

```diff
@@ -89,37 +92,38 @@ def test_download_azureml_dataset(test_output_dirs: OutputFolderForTests) -> Non
     # creation may need access to the dataset.
     with pytest.raises(ValueError) as ex:
         runner.setup()
-    assert ex.value.args[0] == "The model must contain either local_dataset or azure_dataset_id."
+    assert ex.value.args[0] == "Expecting that a dataset is available here."
     runner.project_root = test_output_dirs.root_dir
 
     # Pointing the model to a dataset folder that does not exist should raise an Exception
     fake_folder = runner.project_root / "foo"
     runner.container.local_dataset = fake_folder
     with pytest.raises(FileNotFoundError):
-        runner.mount_or_download_dataset(runner.container.azure_dataset_id, runner.container.local_dataset)
+        runner.download_or_use_existing_dataset(runner.container.azure_dataset_id, runner.container.local_dataset)
 
     # If the local dataset folder exists, mount_or_download_dataset should not do anything.
     fake_folder.mkdir()
-    local_dataset = runner.mount_or_download_dataset(runner.container.azure_dataset_id, runner.container.local_dataset)
+    local_dataset = runner.download_or_use_existing_dataset(runner.container.azure_dataset_id,
+                                                            runner.container.local_dataset)
```

Comment:
Not sure if this is just strange github formatting, but it looks like blank line has randomly been added between the parameters here? (and same in line 115)